tests/test_duplicate_results.py

```diff
@@ -0,0 +1,124 @@
+from collections import defaultdict
+from pathlib import Path
+from typing import Dict, List
+
+import pytest
+import mteb
+
+
+def get_reference_revision(model_name: str) -> str | None:
+    """Get the reference revision for a model from MTEB."""
+    try:
+        model_meta = mteb.get_model_meta(model_name)
+        return model_meta.revision
+    except Exception:
+        return None
+
+
+def find_duplicate_tasks() -> Dict[str, List[Dict]]:
+    """
+    Find all duplicate task results across revisions.
+
+    Returns:
+        A dictionary mapping model names to lists of duplicate task information.
+    """
+    results_folder = Path(__file__).parent.parent / "results"
+    duplicates_found = {}
+
+    for model_folder in sorted(results_folder.glob("*")):
+        if not model_folder.is_dir() or model_folder.name.startswith(".") or model_folder.name == "Human":
+            continue
+
+        model_name = model_folder.name.replace("__", "/")
+        task_to_revisions: Dict[str, List[Dict]] = defaultdict(list)
+
+        # Collect all task files across revisions
+        for revision_folder in model_folder.glob("*"):
+            if not revision_folder.is_dir() or revision_folder.name.startswith("."):
+                continue
+
+            revision = revision_folder.name
+
+            for task_file in revision_folder.glob("*.json"):
+                if task_file.name == "model_meta.json":
+                    continue
+
+                task_name = task_file.stem
+                task_to_revisions[task_name].append({
+                    "revision": revision,
+                    "file_path": task_file,
+                    "revision_folder": revision_folder,
+                })
```

Comment:
I think this would be simpler
```suggestion
                task_to_revisions[task_name].append(revision)
```